File: Proiect/src/main/java/model/Participant.java
package model;

import java.io.Serializable;

public class Participant implements Identifiable<Integer>, Serializable {
    private String name;
    private String email;
    private String phone;
    private Integer ID;

    public Participant(String name, String email, String phone) {
        this.name = name;
        this.email = email;
        this.phone = phone;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "name='" + name + '\'' +
                ", email='" + email + '\'' +
                ", phone='" + phone + '\'' +
                '}';
    }

    @Override
    public Integer getID() {
        return ID;
    }

    @Override
    public void setId(Integer id) {
        ID = id;
    }
}

File: ProiectCsharp/MPP/MPP/model/Participant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MPP.model
{
    internal class Participant : Identifiable<int>
    {
        private int _id;
        private string _name;
        private string _email;
        private string _phone;

        public Participant(int id, string name, string email, string phone)
        {
            Id = id;
            Name = name;
            Email = email;
            Phone = phone;
        }

        public Participant(string name, string email, string phone)
        {
            Name = name;
            Email = email;
            Phone = phone;
        }

        public int Id
        {
            get { return _id; }
            set { _id = value; }
        }

        public string Name
        {
            get { return _name; }
            set { _name = value; }
        }

        public string Email
        {
            get { return _email; }
            set { _email = value; }
        }

        public string Phone
        {
            get { return _phone; }
            set { _phone = value; }
        }

        public override string? ToString()
        {
            return Name + " " + Email + " "+ Phone+  "\n";
        }
    }
}
